stories/LinterMessage.stories.tsx

```diff
@@ -92,6 +92,53 @@ storiesOf('LinterMessage', module).addWithChapters('all variants', {
             />
           ),
         },
+        {
+          title: 'Inlined error',
+          sectionFn: () => (
+            <LinterMessage
+              inline
+              message={createMessage({
+                type: 'error',
+                message: 'The value of &lt;em:id&gt; is invalid',
+                description: [
+                  'The values supplied for &lt;em:id&gt; in the install.rdf file is not a valid UUID string.',
+                ],
+              })}
+            />
+          ),
+        },
+        {
+          title: 'Inlined warning',
+          sectionFn: () => (
+            <LinterMessage
+              inline
+              message={createMessage({
+                type: 'warning',
+                message:
+                  'The manifest contains a dictionary but no id property.',
+                description: [
+                  'A dictionary was found in the manifest, but there was no id set.',
+                ],
+              })}
+            />
+          ),
+        },
+        {
+          title: 'Inlined notice',
```

Comment:
It would help to see one more story that stacks two inline messages so you can see how the triangle should only apply to the first one.

Reply:
:+1: Good idea, we have that in the `CodeView` and `DiffView` components but that would make sense here too